devops/inventory/staging

```diff
@@ -19,3 +19,6 @@ staging
 
 [build]
 build ansible_connection=docker
+
+[test]
+test ansible_connection=docker
```

Comment:
The `test` host is too vaguely named. At first glance, it looks like a throwaway box that won't actually be used for anything. A more descriptive name such as `apptestclient` would be well met.